README.md

```diff
@@ -20,85 +20,74 @@
- * }
- */
-const scatterplotLayer = new ScatterplotLayer({
-  id: 'bart-stations',
-  data: 'https://github.com/uber-common/deck.gl-data/blob/master/website/bart-stations.json',
-  getRadius: d => Math.sqrt(d.entries) / 100,
-  getPosition: d => d.coordinates,
-  getFillColor: [255, 228, 0],
-});
-```
+#### Pure JS
 
-## Using deck.gl with React
+- [Get started](/docs/get-started/using-standalone.md)
+- [Full examples](/examples/get-started/pure-js)
 
-```js
-import DeckGL from 'deck.gl';
+#### React
 
-<DeckGL
-  width="100%"
-  height="100%"
-  initialViewState={{longitude: -122.4, latitude: 37.78, zoom: 8}}
-  controller={true}
-  layers={[scatterplotLayer]} />
-```
+- [Get started](/docs/get-started/using-with-react.md)
+- [Full examples](/examples/get-started/react)
+
+### Python
 
-## Using deck.gl with Pure JS
-
-```js
-import {Deck} from '@deck.gl/core';
-
-const deck = new Deck({
-  width: '100vw',
-  height: '100vh',
-  initialViewState: {
-    longitude: -122.4,
-    latitude: 37.78,
-    zoom: 8
-  },
-  controller: true,
-  layers: [scatterplotLayer]
-});
+```bash
+pip install pydeck
 ```
 
-Minimum setups of end-to-end deck.gl usage is also showcased in the [hello-world examples](./examples/get-started), using both [webpack](https://webpack.js.org/) and [browserify](http://browserify.org/), so you can choose which bundler you prefer or are more familiar with.
+- [Get started](/bindings/pydeck/README.md)
 
-To learn how to use deck.gl through the many examples that come with the deck.gl repo, please clone the latest **release** branch:
+### Third-Party Bindings
+
+- [deckgl-typings](https://github.com/danmarshall/deckgl-typings) (Typescript)
+- [mapdeck](https://symbolixau.github.io/mapdeck/articles/mapdeck.html) (R)
+- [vega-deck.gl](https://github.com/microsoft/SandDance/tree/master/packages/vega-deck.gl) ([Vega](https://vega.github.io/))
 
-```
-git clone -b 7.3-release --single-branch https://github.com/uber/deck.gl.git
-```
 
-For the most up-to-date information, see our [API documentations](http://deck.gl/#/documentation)
+## Learning Resources
 
+* [API documentation](https://deck.gl/#/documentation) for the latest release
+* [Website demos](https://deck.gl/#/examples) with links to source
+* [Interactive playground](https://deck.gl/playground)
+* [deck.gl Codepen demos](https://codepen.io/vis-gl/)
+* [deck.gl Observable demos](https://beta.observablehq.com/@pessimistress)
+* [vis.gl Medium blog](https://medium.com/vis-gl) 
+* [deck.gl Slack workspace](https://join.slack.com/t/deckgl/shared_invite/zt-7oeoqie8-NQqzSp5SLTFMDeNSPxi7eg)
 
 ## Contributing
 
-PRs and bug reports are welcome, and we are actively opening up the deck.gl [roadmap](./dev-docs) to facilitate for external contributors.
+PRs and bug reports are welcome, and we are actively opening up the deck.gl roadmap to facilitate for external contributors. Read the [developement guidelines](/dev-docs) to learn about our dev process.
```

Comment:
- I'd avoid abbreviations: "Pull requests", "development"
- "actively opening up"? Maybe just "deck.gl planning is done openly on GitHub"?